src/html/index.html

```diff
@@ -11,22 +11,26 @@ <h3 data-module="example">{{ message }}</h3>
       <div style="background-color:#4a4a4a; padding:10px; height:auto">
         <button class="c-btn  c-btn--primary">Register</button>
         <button class="c-btn  c-btn--icon  c-btn--icon-huge  c-btn--icon_darkbg">{{sprite("play")}}</button>
-        <button class="o-flex  c-btn  c-btn--icon  c-btn--icon-small  c-btn--icon_darkbg">{{sprite("play")}}
-          <span class="o-flex--text">watch the video</span>
-        </button>
+
+        <div class="c-watch">
+          <button class="c-watch--btn  c-btn  c-btn--icon  c-btn--icon-small  c-btn--icon_darkbg">{{sprite("play")}}
+            <span class="c-watch--text">watch the video</span>
+          </button>
+        </div>
       </div>
 
       <div style="background-color:#fbfbfb; padding:10px; height:auto">
         <button class="c-btn  c-btn--secondary">Register</button>
         <button class="c-btn  c-btn--icon  c-btn--icon-huge  c-btn--icon_lightbg">{{sprite("play")}}</button>
 
-
         <div class="c-share">
-          <button class="c-share--items  c-btn  c-btn--icon c-btn--icon-large  c-btn--icon_lightbg">{{sprite("facebook")}}</button>
-          <button class="c-share--items  c-btn  c-btn--icon c-btn--icon-large  c-btn--icon_lightbg">{{sprite("twitter")}}</button>
-          <button class="c-share--items  c-btn  c-btn--icon c-btn--icon-large  c-btn--icon_lightbg">{{sprite("instagram")}}</button>
+          <span class="c-share__items">share on</span>
+          <button class="c-share__items  c-btn  c-btn--icon c-btn--icon-large  c-btn--icon_lightbg">{{sprite("facebook")}}</button>
```

Comment:
@xavierang let's update these from `button` to `a` tags since they will likely be a link